repo/packages/C/cassandra/1/config.json

```diff
@@ -31,6 +31,23 @@
               "minimum": 0,
               "default": 604800
             },
+            "cpus": {
+              "default": 0.5,
+              "description": "CPU shares to allocate to each Cassandra framework instance.",
+              "type": "number"
+            },
+            "mem": {
+              "default": 512.0,
+              "description": "Memory (MB) to allocate to each Cassandra framework instance.",
+              "minimum": 512.0,
+              "type": "number"
+            },
+            "instances": {
```

Comment:
This should not be configurable.  We only ever want one instance of the scheduler running at a time.  The number of nodes in the cluster is configured by another parameter.